### DISCUSSION TITLE
source/PlayState.hx:298: characters 16-36 : Cannot rebind this method : please use 'dynamic' before method declaration

### DISCUSSION THREAD

**Discussion by AshuraTheEchidna** [FunkinCrew/Funkin#1556] [Compiling help]
i got another error, great, here ill put the file so anybody can tell me whats wrong, and yes, i tried to put dynamic and it didnt worked
[PlayState.txt](https://github.com/ninjamuffin99/Funkin/files/6518880/PlayState.txt)

**Comment by HeyaGlitz** [comment]
could you please specify the method?

**Comment by ghost** [comment]
```haxe
		FlxG.cameras.setDefaultDrawTarget = [camGame];
```

**Comment by AshuraTheEchidna** [comment]
> could you please specify the method?

FlxG.cameras.setDefaultDrawTarget = [camGame];

**Comment by ghost** [comment]
Add `Dynamic ` to the start

**Comment by AshuraTheEchidna** [comment]
> Dynamic

now it says: `source/PlayState.hx:298: characters 11-15 : Missing ;`

**Reply by RevixDev** [reply]
Did u really forget the semicolon on the line before?

**Comment by ghost** [comment]
Use Haxe 4.1.5 or 4.2.1/later

**Comment by AshuraTheEchidna** [comment]
> Use Haxe 4.1.5 or 4.2.1/later

i used 4.1.5 then 4.2.1 and it still didnt worked

**Comment by ghost** [comment]
Reinstall Haxe.

**Comment by AshuraTheEchidna** [comment]
> Reinstall Haxe.

ok, ill tell you if it works or not

**Comment by AshuraTheEchidna** [comment]
> Reinstall Haxe.

it didnt worked, it gave me the same error

**Reply by ghost** [reply]
So its fixed? I still keep getting it aswell

**Comment by RevixDev** [comment]
You forgot the semicolon
